main.go

```diff
@@ -109,3 +117,27 @@ func main() {
 		log.Fatal(err)
 	}
 }
+
+func cleanupPendingBuilds(curfs FileSystem) error {
+	l := log.New(os.Stderr, "[cleanup] ", log.LstdFlags)
+	projects, err := ioutil.ReadDir(cfg.ProjectsPath)
+	if err != nil {
+		return err
+	}
+
+	for _, f := range projects {
```

Comment:
`p` (for "path" or "project") seems more accurate

Reply:
true, changed.